src/main/java/com/avispl/dal/communicator/cisco/CiscoCommunicator.java

```diff
@@ -1569,6 +1588,151 @@ private void populateRoomAnalyticsData(Map<String, String> statistics, List<Adva
         }
     }
 
+    /**
+     * Retrieve Peripherals statistics/controls values. In order to do so - we need to perform a control command
+     * unlike data is normally received with status/configuration xml payloads. Status payload only contains connected
+     * devices data, but in oder to retrieve registered devices also - command action is needed.
+     *
+     * @param statistics             map to set statistics data to
+     * @param controllableProperties list of controls to add controllable properties to
+     * @param status                 device status data fetched from {@link #statusPath}
+     * @param configuration          device configuration data fetched from {@link #configurationPath}
+     * @param valuespace             device valuespace data fetched from {@link #valuespacePath}
```

Comment:
Since there is a javadoc for this method, and the method is new in 1.1, you can add `@since 1.1`